src/components/EmojiPicker/EmojiPicker.tsx

```diff
@@ -221,11 +221,15 @@ function EmojiPicker({viewportOffsetTop}: EmojiPickerProps, ref: ForwardedRef<Em
             shouldEnableNewFocusManagement
             restoreFocusType={CONST.MODAL.RESTORE_FOCUS_TYPE.DELETE}
         >
-            <EmojiPickerMenu
-                onEmojiSelected={selectEmoji}
-                activeEmoji={activeEmoji.current}
-                ref={(el) => (emojiSearchInput.current = el)}
-            />
+            <FocusTrapForModal active={isEmojiPickerVisible}>
+                <View>
```

Comment:
Any reason for wrapping with View?

Reply:
> @situchan, tab key navigation will not work if we don't wrap with view.

what's the root cause?

Reply:
> I have checked other components which uses FocusTrapForModal  and they all wrap the content with View. 

ok but EmojiPickerMenu's parent container is already View:
https://github.com/Expensify/App/blob/e9094967b86d1aa3ba65b64678aa2757a61c0569/src/components/EmojiPicker/EmojiPickerMenu/index.tsx#L308-L315

What's wrong with it?

> Should I look for the RC?

That would be great!

Reply:
I think I found the reason. It's because of forwardRef

https://github.com/Expensify/App/blob/e9094967b86d1aa3ba65b64678aa2757a61c0569/src/components/EmojiPicker/EmojiPickerMenu/index.tsx#L351

If we just return EmojiPickerMenu, it doesn't happen even without wrapping with View.